install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -1973,9 +1973,6 @@ envoy:
       metricRelabelings: ~
     # -- Serve prometheus metrics for cilium-envoy on the configured port
     port: "9964"
-# -- Enable use of the remote node identity.
-# ref: https://docs.cilium.io/en/v1.7/install/upgrade/#configmap-remote-node-identity
-remoteNodeIdentity: true
```

Comment:
I think this might needs the docs to be regenerated with `make -C Documentation update-helm-values`

Reply:
Done